ghostjs-core/src/ghostjs.js

```diff
@@ -84,7 +84,7 @@ class Ghost {
       this.setDriverOpts({parameters: ['-jsconsole']})
     } else if (this.testRunner.match(/chrome/)) {
       const program = spawn(ChromeGhostDriver.path, [], {
-        cwd: root,
+        cwd: process.cwd(),
```

Comment:
No worries, thanks for the PR. It seems that the commit which broke this (the one that set chrome flags), also broke chrome tests running in CI, and early bailed from mocha. 

That's a little more fragile than it probably should be, so I'll look at seeing at what we can do to safeguard. Thanks for the pull request!